src/ontology/mondo-ingest.Makefile

```diff
@@ -163,6 +163,18 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+# TODO: complete
+$(COMPONENTSDIR)/icd11foundation.owl: $(TMPDIR)/icd11foundation_relevant_signature.txt | component-download-icd11foundation.owl
```

Comment:
## Which annotation prop issues in scope for this PR?
- [ ] @matentzn / @twhetzel advise

I created these 3 issues, and added them as checkboxes to the OP, and I set the issue priority as well as added my comment to the side in accordance to how important / timely it is / whether I think it might be in scope for this PR. Personally I don't mind just leaving all of these until after the PR, though. Thoughts?
- #461 (possibly / probably in scope)
- #462 (possibly in scope)
- #463 (possibly / probably not in scope)

Reply:
Thanks, great move. I think you can leave this to a subsequent PR.